bin/compile

```diff
@@ -91,13 +96,18 @@ install_bins() {
     warn_node_engine "$iojs_engine"
     install_iojs "$iojs_engine" "$BUILD_DIR/.heroku/node"
     echo "Using bundled npm version for iojs compatibility: `npm --version`"
+    mcount 'version.iojs.$(node --version)'
```

Comment:
`''` instead of `""` will produce `version.iojs.$(node --version)` AFAIK. PS: This is why I suggest `""` over mixing the two, except for the cases where you *really* need it to be `''`.